### PR TITLE
Add more detail attributes for certificate-error

### PR DESCRIPTION
**Pull request by darkdh**
We can extract more detail from certificate when encountering error.

### CODE REVIEW

**Comment by zeke** [comment]
Started here: https://github.com/brave/electron/pull/25

File: atom/common/native_mate_converters/net_converter.cc [hunk 1]
```diff
@@ -46,6 +46,12 @@ v8::Local<v8::Value> Converter<scoped_refptr<net::X509Certificate>>::ToV8(
                                    encoded_data.size()).ToLocalChecked();
   dict.Set("data", buffer);
   dict.Set("issuerName", val->issuer().GetDisplayName());
+  dict.Set("subjectName", val->subject().GetDisplayName());
+  dict.Set("serialNumber", val->serial_number());
+  dict.Set("validStart", val->valid_start().ToDoubleT());
+  dict.Set("validExpiry", val->valid_expiry().ToDoubleT());
+  dict.Set("fingerprint", net::HashValue(val->fingerprint()).ToString());
```
Comment (deepak1556) [inline atom/common/native_mate_converters/net_converter.cc]: `fingertprint` method is removed in future releases, related CL https://codereview.chromium.org/2000503002

Comment (darkdh) [inline atom/common/native_mate_converters/net_converter.cc]: It is replaced by `CalculateFingerprint256()` in 02e89d6

**Comment by zcbenz** [comment]
👍

**Pull request merged**